docs/07. media library/02. integrating in your module.md

```diff
@@ -0,0 +1,293 @@
+# How to integrate the Fork CMS MediaLibrary Module into a custom Fork CMS Module?
+
+> The example below shows you how easy it is to add media groups to your custom module. We show you how to add an "images" and "trailers" MediaGroup.
+
+Overview:
+* [Integration in Backend](#integration-in-backend)
+* [Integration in Frontend](#integration-in-frontend)
+* [Database changes](#database-changes)
+
+## In your custom Entity
+
+```php
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroup;
+```
+
+```php
+/**
+ * @var MediaGroup
+ *
+ * @ORM\OneToOne(
+ *      targetEntity="Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroup",
+ *      cascade="persist",
+ *      orphanRemoval=true
+ * )
+ * @ORM\JoinColumn(
+ *      name="mediaGroupId",
+ *      referencedColumnName="id",
+ *      onDelete="cascade"
+ * )
+ */
+protected $group;
+```
+
+```php
+/**
+ * Gets the value of group.
+ *
+ * @return MediaGroup
+ */
+public function getGroup()
+{
+    return $this->group;
+}
+```
+
+You will also need to add set `$this->group = $group` in your constructor (and `create`) and `update` method.
+
+
+### In your Symfony FormType class
+
+```php
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroupType;
+```
+
+```php
+$builder->add(
+    'mediaGroup',
+    MediaGroupType::class,
+    [
+        'label' => 'lbl.MediaConnected',
+    ]
+)
+```
+> You can replace `lbl.MediaConnected` by your own backend label, f.e.: 'Screenshots', 'Trailers', 'Preview images', ...
+
+Example: `Backend/Modules/MediaGalleries/Domain/MediaGallery/MediaGalleryType.php#L92`
+
+### In your Create Command class
+
+```php
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroup;
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\Type as MediaGroupType;
```

Comment:
@WouterSioen there is no documentation for the commandbus afaik,after the release of fork 5 maybe it would be a good idea to focus on getting the documentation a bit more up to date